website/docs/tutorials/guides/deployment.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: Deployment
+---
+
+# Deployment
+
+Protostar `>=0.2.1` supports deploying smart contracts to a given network with the `protostar deploy` command. It has similar interface to `starknet deploy` command. [Read the CLI reference for the deploy command](/docs/cli-reference#deploy) to learn more about all supported arguments.
```

Comment:
I think the information `>=0.2.1` is unnecesary, it's place is in changelog, not in a guide.